tabular/src/autogluon/tabular/configs/hyperparameter_configs.py

```diff
@@ -79,23 +79,9 @@
         ],
         'CAT': {},
         'XGB': {},
-        # 'FASTAI': {},  # FIXME: FASTAI is extremely slow if trained alongside TextNN due to forkserver.
-        'RF': [
-            {'criterion': 'gini', 'ag_args': {'name_suffix': 'Gini', 'problem_types': ['binary', 'multiclass']}},
-            {'criterion': 'entropy', 'ag_args': {'name_suffix': 'Entr', 'problem_types': ['binary', 'multiclass']}},
-            {'criterion': 'mse', 'ag_args': {'name_suffix': 'MSE', 'problem_types': ['regression']}},
-        ],
-        'XT': [
-            {'criterion': 'gini', 'ag_args': {'name_suffix': 'Gini', 'problem_types': ['binary', 'multiclass']}},
-            {'criterion': 'entropy', 'ag_args': {'name_suffix': 'Entr', 'problem_types': ['binary', 'multiclass']}},
-            {'criterion': 'mse', 'ag_args': {'name_suffix': 'MSE', 'problem_types': ['regression']}},
-        ],
-        'KNN': [
-            {'weights': 'uniform', 'ag_args': {'name_suffix': 'Unif'}},
-            {'weights': 'distance', 'ag_args': {'name_suffix': 'Dist'}},
-        ],
-        'TEXT_NN_V1': ['ELECTRA_BASE'],
+        # 'FASTAI': {}, FastAI gets killed if the dataset is large (400K rows).
```

Comment:
For the issue I experienced, it may not be related to forkserver code because I disabled that. It's just that the process gots killed when I'm training the model on the 430K dataset.